CHANGELOG.md

```diff
@@ -77,6 +78,33 @@ title: Changelog
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 3.11.0
+
+### Change
+
+- jwt-auth plugin no longer requires a private_key to be uploaded. [#11597](https://github.com/apache/apisix/pull/11597)
+- rewrite hmac-auth plugin for usability [#11581](https://github.com/apache/apisix/pull/11581)
+
+### Plugins
+
+- allow configuring keepalive_timeout [#11611](https://github.com/apache/apisix/pull/11611)
+- add plugin attach-consmer-label [#11604](https://github.com/apache/apisix/pull/11604)
```

Comment:
Where is the preceding PR that adds the "additional headers"?

<img width="919" alt="image" src="https://github.com/user-attachments/assets/44c14b55-fa7a-486f-b51d-4f489483a825">

Reply:
here: https://github.com/apache/apisix/pull/11601

and I have mentioned that in the changelog here:
https://github.com/apache/apisix/pull/11620/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR106